drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -213,15 +236,15 @@ def ce_assessment_factory(create: true)
 
     return ce_assessment if ce_assessment.present? || !create
 
-    self.ce_assessment = enrollment_factory.assessments.build(user_id: custom_assessment&.user_id)
+    self.ce_assessment = enrollment_factory.assessments.build
```

Comment:
These all were redundant because assign_metadata handles setting the user_id based on the current user that is submitting.